app/views/home/index.html.erb

```diff
@@ -159,25 +159,25 @@
     <h2 class="alt-h2 mt-6 text-center">You're in good company.</h2>
 
     <div class="clearfix mt-6 pt-6 mb-4 border-top">
-      <div class="col-sm-6 float-left mb-4 mb-md-0 pr-sm-3 pr-lg-4">
+      <div class="col-sm-6 float-left pl-sm-3 pl-lg-4">
         <div>
           <div class="d-table-cell v-align-top pr-3">
-            <img src="https://api.adorable.io/avatars/285/ossfriday.png" alt="" width="50" height="50" style="border-radius:25px;">
+            <img src="https://avatars2.githubusercontent.com/u/9155255?v=3&s=100" alt="" width="50" height="50" style="border-radius:25px;">
           </div>
           <blockquote class="d-table-cell v-align-top text-gray">
-            <p class="alt-mono-font alt-text-small">“Quote from someone that made their first contribution to open source.”</p>
-            <footer class="alt-text-small">Jane Smith, Columbus OH</footer>
+            <p class="alt-mono-font alt-text-small">Contributing to open source not only allows businesses of every size to take advantage and build upon the talents of others, it’s also an avenue for those new to coding to acclimate themselves to the development process in ways that allow them to practice, experiment, and showcase themselves as valuable and contributing members of the community.</p>
+            <footer class="alt-text-small">Kim Crayton (<a href="https://github.com/kmcrayton7">@kmcrayton7</a>), Community Engineer, Writer, Founder</footer>
```

Comment:
@kmcrayton7 Just a heads up that your quote is being used here. Let me know either here or at mikemcquaid@github.com if you have any problem with that. Ignore this if you're 👍. Thanks!